x-pack/plugins/security_solution/server/client/client.ts

```diff
@@ -19,6 +24,7 @@ export class AppClient {
   constructor(spaceId: string, config: ConfigType, kibanaVersion: string, kibanaBranch: string) {
     const configuredSignalsIndex = config.signalsIndex;
 
+    this.alertsIndex = `${DEFAULT_ALERTS_INDEX}-${spaceId}`;
```

Comment:
Correct, since we're not in 7.x it's possible (likely) that the user doesn't have the 7.x `.siem-signals` alias, but they will have the alerts one. In fact, this was already done for [previous risk score code](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/utils/risk_score_modules.ts#L36), and I should consolidate those functions.